challenge_serving_drinks/src/challenge_serving_drinks/serving_drinks.py

```diff
@@ -0,0 +1,82 @@
+# ROS
+import smach
+
+# TU/e Robotics
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+
+from robot_skills.classification_result import ClassificationResult
+from robot_skills.robot import Robot
+from robocup_knowledge import knowledge_loader
+
+# Challenge serving drinks
+from .drive_in import DriveIn
+from .serve_one_drink import ServeOneDrink
+
+# Knowledge
+CHALLENGE_KNOWLEDGE = knowledge_loader.load_knowledge("challenge_serving_drinks")
+
+
+class ServingDrinks(smach.StateMachine):
+    """ State machine for 'Serving Drinks' challenge.
+
+    """
+
+    def __init__(self, robot):
+        # type: (Robot) -> str
+        """ Initialization method
+
+        :param robot: robot api object
+        """
+
+        smach.StateMachine.__init__(self, outcomes=["succeeded", "failed", "aborted"])
+
+        # Designators
+        bar_designator = ds.EdEntityDesignator(robot=robot, id=CHALLENGE_KNOWLEDGE.bar_id, name='bar_des')
+        room_designator = ds.EdEntityDesignator(robot=robot, id=CHALLENGE_KNOWLEDGE.room_id, name='room_des')
+
+        objects_list_des = ds.VariableDesignator(resolve_type=[ClassificationResult], name='objects_list_des')
+        unav_drink_des = ds.VariableDesignator(resolve_type=str, name='unav_drink_str_des')
+
+        with self:
+            smach.StateMachine.add(
+                "DRIVE_IN",
```

Comment:
Also style fixes in this file.

Reply:
Will be reflected in the new PR (styling)